src/Associations.cc

```diff
@@ -85,6 +89,33 @@ void Associations::setCommonTangentPoint(lsst::afw::geom::Point2D const &commonT
     for (auto &ccdImage : ccdImageList) ccdImage->setCommonTangentPoint(_commonTangentPoint);
 }
 
+lsst::sphgeom::Circle Associations::computeBoundingCircle() const {
+    // Compute the frame on the common tangent plane that contains all input images.
+    Frame tangentPlaneFrame;
+
+    for (auto const &ccdImage : ccdImageList) {
+        Frame CTPFrame = ccdImage->getPixelToCommonTangentPlane()->apply(ccdImage->getImageFrame(), false);
+        if (tangentPlaneFrame.getArea() == 0)
+            tangentPlaneFrame = CTPFrame;
+        else
+            tangentPlaneFrame += CTPFrame;
+    }
+
+    // Convert tangent plane coordinates to RaDec.
+    AstrometryTransformLinear identity;
+    TanPixelToRaDec commonTangentPlaneToRaDec(identity, _commonTangentPoint);
+    Frame raDecFrame = commonTangentPlaneToRaDec.apply(tangentPlaneFrame, false);
+
```

Comment:
I'll have to trust based on the tests that this works, because I have no experience with the wcs `Frame`s.

Reply:
The `Frame` in this case, is actually a jointcal object. I want to replace it with `lsst.geom.Box2D` at some point (DM-4044).